detekt-config.yml

```diff
@@ -367,8 +369,12 @@ potential-bugs:
     active: true
   UnreachableCode:
     active: true
+  MapGetWithNotNullAssertionOperator:
+    active: true
+    excludes: [ '**/test/**', '**/*Test.kt', '**/*Spec.kt' ]
   UnsafeCallOnNullableType:
-    active: false
+    active: true
+    excludes: ['**/test/**', '**/*Test.kt', '**/*Spec.kt']
```

Comment:
nit: Why IAM is not using the detekt-config file from Shared Build-config, maybe it's better to update the shared build-config one (to reuse this update for other projects as well)?

Reply:
This is very specific for our usage.
We can improve it but that will have effect on other users of the config